docs/ClusterSetup.md

```diff
@@ -221,6 +224,7 @@ zookeeperServers=zk1.us-west.example.com:2181,zk2.us-west.example.com:2181,zk3.u
 globalZookeeperServers=zk1.us-west.example.com:2184,zk2.us-west.example.com:2184,zk3.us-west.example.com:2184
 
 clusterName=us-west
+statusFilePath=broker-status-file
```

Comment:
This setting did not work for me, is there a default provided in the code? Or does this setting rely on being set in the file? If the latter, might be worth providing a sane default in the code so the broker does not error if the user misses this?